src/JavaSession1/ReplaceString.java

```diff
@@ -0,0 +1,28 @@
+package JavaSession1;/*Question 1: Write a program to replace a substring inside a string with other string ?*/
+
+import java.util.Scanner;
+
+public class ReplaceString {
+
+    private static String replaceString(String orignalString, String replacementString, String newString)
+    {
+        return orignalString.replace(replacementString,newString);
+    }
```

Comment:
I think there is no need to write this method. Since there is only one line here which is using the existing standard logic. Please format code always. 
m1(){
}